toolchain/lowering/lowering_context.h

```diff
@@ -34,14 +35,18 @@ class LoweringContext {
   auto GetType(SemanticsTypeId type_id) -> llvm::Type* {
     // InvalidType should not be passed in.
     if (type_id == SemanticsTypeId::TypeType) {
-      // TODO: Handle TypeType. Currently using a place holder
-      // as a workaround.
-      return llvm::Type::getInt1Ty(llvm_context());
+      // `type` is represented as `{}`.
```

Comment:
```suggestion
      // `type` is represented as an empty StructType.
```

{} is meaningful in Carbon, and `type` is Carbon, so looking to disambiguate versus LLVM IR (because `type` _could_ be a Carbon `{}` if so chosen). Not that this phrasing is great either.

Reply:
Hopefully this phrasing is a bit clearer.